NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# 37 ds delete category

#### Changes Made
1. Added file `categoryDelete.js` to `src/components/Categories` directory.
1. Modified file `categoryManager.js` to include `deleteCategory` function.
2. Modified `ApplicationViews.Js` to include routing for delete confirmation view

​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout 37_DS_delete_category
    ```
4. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
    > when a logged in user selects the delete link in the category list they are redirected to a confirmation page with the corresponding category name. when the delete button is clicked the data is removed and the user is redirected back to the category list
6. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
